### DISCUSSION TITLE
Welcome to j471n.in Discussions!

### DISCUSSION THREAD

**Discussion by j471n** [j471n/j471n.in#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SpandanSehgal** [comment]
Hi @j471n looking forward to have a great discussion and great time with you and all the community members.

I already have a question :
How did you name your site? 
J471n 
I.e. how did this name strike you. 
Fun fact I've also changed my portfolio name by seeing your name 😆

**Reply by j471n** [reply]
Hi @SpandyBoss, 

There were two reasons to put 47 in my name-

1. Because of [Agent 47](https://en.wikipedia.org/wiki/Agent_47)
2. There is an Indian Rapper called [Fotty Seven](https://www.youtube.com/channel/UCF30CDaqpyrfJD51j-ZP8kg). I was listening to him a lot at that time. 

and in j471n, 1 is  just a replacement for letter `i`

 
P.S.: 4 and 7 also stands for letter A and T respectively.

**Reply by SpandanSehgal** [reply]
Ohk thanks @j471n 😊
